django_projects/settings.py

```diff
@@ -54,6 +55,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware'
```

Comment:
According to https://stackoverflow.com/questions/43357687/django-python-rest-framework-no-access-control-allow-origin-header-is-present, it should be first in list. Check my changes in docker branch.

Reply:
It worked for me like this, should we just scrap this branch then if you made the changes on your branch?

Reply:
Sure. Wasn't sure if we wanted to merge Docker changes yet. We can talk tonight.